.github/workflows/whitespace-linter

```diff
@@ -0,0 +1,16 @@
+name: Whitespace linter
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  whitespace_lint:
+    runs-on: ubuntu-latest
+    if: contains(github.event.comment.body, '/wslint')
+    steps:
+    - uses: actions/checkout@v2
+    - name: Whitespace linter
+      uses: hexrabbit/test-action@v0.0.1
```

Comment:
This action doesn't exist. I believe you want this:
```suggestion
      uses: hexrabbit/whitespace-lint@v0.0.1
```
The documentation is not quite correct here.

Reply:
Thanks, fixed.